config/clusters/nasa-veda/support.values.yaml

```
@@ -22,6 +22,15 @@ grafana:
         hosts:
           - grafana.nasa-veda.2i2c.cloud
 
+redirects:
+  rules:
+    # nasa-veda.2i2c.cloud was the old URL
+    # Moved due to https://github.com/2i2c-org/infrastructure/issues/3029
+    - from: nasa-veda.2i2c.cloud
+      to: veda.2i2c.cloud
+    - from: staging.nasa-veda.2i2c.cloud
+      to: staging.veda.2i2c.cloud
```

```suggestion
      to: staging.veda.2i2c.cloud
    - from: grafana.nasa-veda.2i2c.cloud
      to: grafana.veda.2i2c.cloud
```

Let's get grafana updated as well (and the associated callback URL for the oauth2 application)